app/views/idv/sessions/index.html.slim

```diff
@@ -24,7 +24,7 @@
       class: 'block col-12 field', input_html: { 'aria-labelledby': 'label-dob' }
   h2#label-ssn.heading = t('idv.titles.session.ssn')
   .mb4.sm-col-5
-    = f.input :ssn, label: false, required: true, pattern: '[0-9]*',
+    = f.input :ssn, label: false, required: true, pattern: '^\d{3}-?\d{2}-?\d{4}$',
```

Comment:
Presumably in some idv helper or controller class.  However, I'm happy for this PR to stay focused if you would rather just land this one now.   My main concern is that I'd like to see comments document why this exact pattern is the right one to use.   Also, I wonder if we can create a test that will fail with the old pattern and pass with the new one?